server/server.js

```diff
@@ -4,10 +4,40 @@ const app = express()
 
 const port = Number(process.argv[2]) || 8000
 
-const staticFiles = express.static(`${__dirname}/www`)
+const staticFiles = express.static('${__dirname}/www')
```

Comment:
when we pass varialbes (like `__dirname` or `port`) into strings like this `${__dirname}` we need to create that string with tick marks, not quotes. u have a string with quotes like this:
```js
'${__dirname}/www'
```
but they should be tick marks like this:
```js
`${__dirname}/www`
```